backup.ps1

```diff
@@ -334,7 +336,15 @@ function Send-Email {
         # create a temporary error log to log errors; can't write to the same file that Send-MailMessage is reading
         $temp_error_log = $ErrorLog + "_temp"
 
-        Send-MailMessage @ResticEmailConfig -From $ResticEmailFrom -To $ResticEmailTo -Credential $credentials -Subject $subject -Body $body @attachments 3>&1 2>> $temp_error_log
+        #sender ([MimeKit.MailboxAddress] http://www.mimekit.net/docs/html/T_MimeKit_MailboxAddress.htm, required)
+        $From = [MimeKit.MailboxAddress]$ResticEmailFrom;
+        $RecipientList = [MimeKit.InternetAddressList]::new();
+        $RecipientList.Add([MimeKit.InternetAddress]$ResticEmailTo);
+        $Port = 465
+        #authentication ([System.Management.Automation.PSCredential], optional)
+        $Credential = [System.Management.Automation.PSCredential]::new($ResticEmailFrom, (ConvertTo-SecureString -String $ResticEmailPassword -AsPlainText -Force));
```

Comment:
`$ResticEmailFrom` is not necessarily the username. There is `$ResticEmailUsername` in `secrets.ps1`